twitchpopups.htm

```diff
@@ -1,30 +1,11 @@
 <!DOCTYPE HTML>
 <html>
 <head>
-
-    <script type="text/javascript">

```

Comment:
This is aimed at people who likely have limited code knowledge. The config is up here at the top so that it's easy to edit